integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -158,19 +166,19 @@ export class OpfResourceLoaderService extends ScriptLoader {
       })),
     ];
 
-    return new Promise((resolve) => {
+    return new Promise((resolve, reject) => {
```

Comment:
adding 'reject' callback when an error occurred, it is then caught in caller function `loadAndRunScript`.
the goal is to not display html in UI if one of the resources (JS or CSS) fails to load.